### PR TITLE
4.12 install doc updates from IBM P

### PR DESCRIPTION
**Pull request by alishaIBM**
Version(s): 4.12

Links : [Deprecation of hardware (RHEL 9 based RHCOS)](https://issues.redhat.com/browse/MULTIARCH-3176)

Link to docs preview :
https://54260--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/installing-ibm-power.html#minimum-ibm-power-system-requirements_installing-ibm-power
https://54260--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/installing-ibm-power.html#recommended-ibm-power-system-requirements_installing-ibm-power

Reference : 
[MULTIARCH-3176] Add deprecation note to IBM Z installation [#53985](https://github.com/openshift/openshift-docs/pull/53985)

### CODE REVIEW

**Review by tmalove — COMMENTED** [review]
@alishaIBM  Just checking to make sure that you meant that the 'IBM Power 9' models are deprecated because you removed 'IBM Power 8' from the content Otherwise looks fine.

**Comment by alishaIBM** [comment]
> @alishaIBM Just checking to make sure that you meant that the 'IBM Power 9' models are deprecated because you removed 'IBM Power 8' from the content Otherwise looks fine.

@tmalove have updated the PR. Thanks for pointing out.

File: modules/minimum-ibm-power-system-requirements.adoc [hunk 1]
```diff
@@ -9,7 +9,12 @@
 
 You can install {product-title} version {product-version} on the following IBM hardware:
 
-* IBM Power8, Power9, or Power10 processor-based systems
+* IBM Power9 or Power10 processor-based systems
+
+[NOTE]
+====
+Support for {op-system} functionality for IBM POWER8 all models, IBM POWER9 AC922, IBM POWER9 IC922, and IBM POWER9 LC922 is deprecated. These hardware models remain fully supported in {product-title} 4.12. However, Red Hat recommends that you use later hardware models.
```
Comment (stevsmit) [inline modules/minimum-ibm-power-system-requirements.adoc:16]: Could we maybe slightly revise this note? It's a little confusing as-is. Maybe something like...

Support for {op-system} functionality for all IBM POWER8 models, IBM POWER9 AC922, IBM POWER9 IC922, and IBM POWER9 LC922, is deprecated. These hardware models remain fully supported in {product-title} 4.12. However, Red Hat recommends that you use later hardware models.

Comment (alishaIBM) [reply modules/minimum-ibm-power-system-requirements.adoc:16]: Ack

**Pull request merged**